lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java

```diff
@@ -0,0 +1,33 @@
+package com.kmwllc.lucille.core;
+
+import static org.junit.Assert.assertEquals;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.List;
+import org.junit.Test;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class FileToFileIngestIT {
+
+  @Test
+  public void fullIT() throws Exception {
+    File f = null;
+    try {
+      Config config = ConfigFactory.parseFile(new File("src/test/resources/FileToFileIngestIT/file-to-file-example.conf"));
+      Runner.runInTestMode(config);
```

Comment:
we're cleaning up the file at the end of the test, which is nice and self-contained
why not also delete the file at the beginning in case it exists, rather than relying on a maven pre-integration phase?